# *CanSeer*

The RNA-seq gene expression datasets used in case studies 1, 2, and 3 were obtained from The Cancer Genome Atlas (TCGA) PanCancer Atlas [1] using Genomic Data Commons (GDC) portal [2].


Genetic alterations employed in case studies 1, 2, and 3 include copy number alterations, somatic mutations, and genomic structural variations, which were acquired from cBioPortal [3], [4].


"Sample Data" was used as an input to the code for data pre-processing, and normalization.

1. [1]	Z. Wang, M. A. Jensen, and J. C. Zenklusen, “A practical guide to The Cancer Genome Atlas (TCGA),” in Methods in Molecular Biology, vol. 1418, Humana Press Inc., 2016, pp. 111–141.
2. [2]	M. A. Jensen, V. Ferretti, R. L. Grossman, and L. M. Staudt, “The NCI Genomic Data Commons as an engine for precision medicine,” Blood, vol. 130, no. 4. American Society of Hematology, pp. 453–459, 27-Jul-2017.
3. [3]	J. Gao et al., “Abstract 923: The cBioPortal for Cancer Genomics: An intuitive open-source platform for exploration, analysis and visualization of cancer genomics data,” in Cancer Research, 2018, vol. 78, no. 13 Supplement, pp. 923–923.
4. [4]	E. Cerami et al., “The cBio Cancer Genomics Portal: An Open Platform for Exploring Multidimensional Cancer Genomics Data,” Cancer Discov., vol. 2, no. 5, pp. 401–404, May 2012.
